public/src/components/admin/MarketState.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { Metadata } from "grpc-web-client";
+import { showNotif, showErrorNotif, closeNotifs } from "../../utils";
+import { DalalActionService } from "../../../proto_build/DalalMessage_pb_service";
+import { OpenMarketRequest } from "../../../proto_build/actions/OpenMarket_pb";
+import { CloseMarketRequest } from "../../../proto_build/actions/CloseMarket_pb";
+
+export interface MarketStateProps{
+    sessionMd: Metadata,
+    isMarketOpen: boolean
+}
+export interface MarketStateState{
+    option: boolean
```

Comment:
use some other variable name than the generic `option`.

Reply:
am using same variable for close as well as open market. So felt generic name would suit better.
Any other name suggestions?